charts/civil-ccd/values.aat.template.yaml

```diff
@@ -23,17 +25,64 @@ ccd:
   ccd-data-store-api:
     java:
       ingressHost: civil-ccd-data-store-staging-aat.aat.platform.hmcts.net
+      environment:
+        DEFINITION_STORE_DB_PASSWORD: "{{ .Values.postgresql.auth.postgresPassword}}"
+        DEFINITION_STORE_DB_OPTIONS: "?stringtype=unspecified"
+      secrets:
+        STORAGE_ACCOUNT_NAME:
+          secretRef: storage-secret-{{ .Release.Name }}-blobstorage
+          key: storageAccountName
+          disabled: true
+        STORAGE_ACCOUNT_KEY:
+          secretRef: storage-secret-{{ .Release.Name }}-blobstorage
+          key: accessKey
+          disabled: true
+        DEFINITION_STORE_DB_PASSWORD:
```

Comment:
In data store pod, why do we need to explicitly say DEFINITION_STORE_DB_PASSWORD disabled as true. I have seen ccd chart it looks only for DATA_STORE_DB_PASSWORD https://github.com/hmcts/chart-ccd/blob/master/ccd/values.yaml#L280C7-L280C29